contracts/button/Cargo.toml

```diff
@@ -19,12 +19,12 @@ access_control = { path = "../access_control", default-features = false, feature
 game_token = { path = "../game_token", default-features = false, features = ["ink-as-dependency"] }
```

Comment:
all these should likely be in the `std` as well

Reply:
sure, added